test/stdin.jl

```diff
@@ -1,6 +1,7 @@
 using DataTables
```

Comment:
I think you can remove this file too, as well as `stdin.sh`. If that needs to be tested, it should go to CSV.jl.